systems/notification/mailer/pkg/server/mailer.go

```diff
@@ -0,0 +1,129 @@
+package server
+
+import (
+	"bytes"
+	"context"
+	"errors"
+	"html/template"
+	"net/smtp"
+	"strconv"
+	"strings"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+
+	"github.com/ukama/ukama/systems/common/config"
+	"github.com/ukama/ukama/systems/common/grpc"
+	"github.com/ukama/ukama/systems/common/uuid"
+	pb "github.com/ukama/ukama/systems/notification/mailer/pb/gen"
+	"github.com/ukama/ukama/systems/notification/mailer/pkg/db"
+)
+
+type EmailData struct {
+	To      []string 
+	Subject string
+	Body    string
+	Values  map[string]interface{}
+}
+
+type MaillingServer struct {
+	maillingRepoRepo db.MaillingRepo
+	pb.UnimplementedMaillingServiceServer
+	mailer *config.Mailer
+}
+
+func NewMaillingServer(maillingRepoRepo db.MaillingRepo, mail *config.Mailer) *MaillingServer {
```

Comment:
*Mailing Need to fix in at most of method name, proto and struct .

Reply:
I have updated all the names to "mailer" throughout the code.